master-folder/ttt/models.py

```diff
@@ -4,10 +4,13 @@
 
 board = []
 
-for row in range(3):
-    board.append([])
-    for column in range(3):
-        board[row].append('.')
+def create_board():
+    global board
```

Comment:
When calling a global variable inside a function you have to declare that you want it to look for that variable on the global scope. Otherwise it will look for it inside the function only. I thought we were going to look at developing the game logic further once Sophie had finished her tutorial. It won't make any difference as it all functions fine and the plan is to refactor it further to make it use classes so the variables won't be global but this was just a quick refactor to make it work more how we wanted it.